### PR TITLE
Feature/init refactoring

### PR DESCRIPTION
**Pull request by premaddx**

### CODE REVIEW

File: PG/README.md [hunk 1]
```diff
@@ -5,10 +5,10 @@ Node version 10.9.0 was used for this SDK.
```
Comment (kshirish) [inline PG/README.md:5]:
```js
const PaymentGateway = require('cashfree-sdk').PG;

// Instantiate Cashfree Payment Gateway
const PG = new PaymentGateway({
    env: '<ENVIRONMENT>',
    apiVersion: '<API_VERSION>',
    appId: '<YOUR_APP_ID>',
    secretKey: '<YOUR_SECRET_KEY>',
});
```

@premaddx Here, `PG` variable below where does it point to ... `PaymentGateway` or `PG` variable above? 

```js
PG.Orders.GetLink({
    orderId: '<ORDER_ID>',
}).then(data => console.log(data)).catch(error => console.error(error));
```

File: PG/Orders/index.js [hunk 2]
```diff
@@ -10,135 +9,128 @@ const {
...
-        return {
-            status: 'ERROR',
-            reason: e && e.message || 'Something went wrong',
-        };
-    }
-};
+function Orders() {
+    const { baseUrl: hostname, appId, secretKey } = this;
+    return {
+        CreateOrders: async (data = {}) => {
```
Comment (kshirish) [inline PG/Orders/index.js:11]: Except modules and constructors, all methods should be camelCase.

File: PG/README.md (hunk 1 above)
Comment (premaddx) [reply PG/README.md:5]: PG variable above.

File: PG/Orders/index.js (hunk 2 above)
Comment (premaddx) [reply PG/Orders/index.js:11]: I followed the nomenclature from Payouts. I will have to ask to Raj for this change.

File: PG/README.md (hunk 1 above)
Comment (kshirish) [reply PG/README.md:5]: @premaddx 

```js
import React, {Component} from 'react';

// Here, we should not be creating the variable with the same name as the imported one right?
const Component = new SomeClass({ ... })

// This is confusing
Component.someMethod()
```

**Pull request merged**